aspnetcore/tutorials/signalr-typescript-webpack.md

```diff
@@ -191,10 +191,10 @@ The app currently displays a simple form to send messages. Nothing happens when
 1. Execute the following command at the project root:
 
     ```console
-    npm install @aspnet/signalr
+    npm install @micrsofot/signalr
```

Comment:
```suggestion
    npm install @microsoft/signalr
```

Reply:
I also think we need to update the following since it shows up in the doc as a snippet.

https://github.com/aspnet/AspNetCore.Docs/blob/1b86849aaaf162b11819c8000ab21f2f2b1ee29f/aspnetcore/tutorials/signalr-typescript-webpack/sample/3.x/snippets/index2.ts#L4

Technically the [package.json](https://github.com/aspnet/AspNetCore.Docs/blob/1b86849aaaf162b11819c8000ab21f2f2b1ee29f/aspnetcore/tutorials/signalr-typescript-webpack/sample/3.x/package.json#L26) for that sample should also be updated to match, but at least that doesn't show up directly in the doc output.